code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -19,7 +20,7 @@
 	var/portal_duration = 5 MINUTES
 	/// How long a target remains in the Syndicate jail.
 	var/prison_time = 4 MINUTES
-	/// List of items a target can get randomly after their return.
+	/// How long an antagonist target remains in the Syndicate jail.
```

Comment:
It appears you deleted a comment here, can you put this comment back and put your new comment above your define?